qemu/tests/disable_win_update.py

```diff
@@ -11,20 +23,33 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
+    def disable_win_service(session, scname):
+        """
+        :param session: VM session.
+        :param scname: Service name.
+
+        :return: return True if scname has been disabled.
+        """
+        session.sendline("sc config %s start= disabled" % scname)
```

Comment:
From help:
A space is required between an option and its value (for example, type= own. If the space is omitted, the operation will fail.